file-formats/ntta/examples/z_aff_example_ntta.ntta.json

```
@@ -0,0 +1,17 @@
+{
+    "header": {
+      "abapLanguageVersion": "standard"
```

Also here, I suggest 
```suggestion
      "description": "Example object",
      "originalLanguage" "en"
```